crates/uv-interpreter/src/interpreter.rs

```diff
@@ -284,32 +306,103 @@ impl Interpreter {
         (self.implementation_major(), self.implementation_minor())
     }
 
+    /// Returns the implementation name (e.g., `CPython` or `PyPy`).
     pub fn implementation_name(&self) -> &str {
         &self.markers.implementation_name
     }
+
+    /// Return the `sys.base_exec_prefix` path for this Python interpreter.
     pub fn base_exec_prefix(&self) -> &Path {
         &self.base_exec_prefix
     }
+
+    /// Return the `sys.base_prefix` path for this Python interpreter.
     pub fn base_prefix(&self) -> &Path {
         &self.base_prefix
     }
 
-    /// `sysconfig.get_path("stdlib")`
-    pub fn stdlib(&self) -> &Path {
-        &self.stdlib
-    }
+    /// Return the `sys.executable` path for this Python interpreter.
     pub fn sys_executable(&self) -> &Path {
         &self.sys_executable
     }
+
+    /// Return the `purelib` path for this Python interpreter, as returned by `sysconfig.get_paths()`.
+    pub fn purelib(&self) -> &Path {
+        &self.sysconfig.purelib
+    }
+
+    /// Return the `platlib` path for this Python interpreter, as returned by `sysconfig.get_paths()`.
+    pub fn platlib(&self) -> &Path {
+        &self.sysconfig.platlib
+    }
+
+    /// Return the `scripts` path for this Python interpreter, as returned by `sysconfig.get_paths()`.
+    pub fn scripts(&self) -> &Path {
+        &self.sysconfig.scripts
+    }
+
+    /// Return the `data` path for this Python interpreter, as returned by `sysconfig.get_paths()`.
+    pub fn data(&self) -> &Path {
+        &self.sysconfig.data
+    }
+
+    /// Return the `include` path for this Python interpreter, as returned by `sysconfig.get_paths()`.
+    pub fn include(&self) -> &Path {
+        &self.sysconfig.include
+    }
+
+    /// Return the `stdlib` path for this Python interpreter, as returned by `sysconfig.get_paths()`.
+    pub fn stdlib(&self) -> &Path {
+        &self.sysconfig.stdlib
+    }
+
+    /// Return the [`Target`] environment used to install wheels into this interpreter.
+    pub fn target(&self) -> Target {
+        Target {
+            python_version: self.python_tuple(),
+            sys_executable: self.sys_executable().to_path_buf(),
+            purelib: self.purelib().to_path_buf(),
+            platlib: self.platlib().to_path_buf(),
+            scripts: self.scripts().to_path_buf(),
+            data: self.data().to_path_buf(),
+            // Detect if this is a PEP 405-compliant virtual environment.
+            // See: https://github.com/pypa/pip/blob/0ad4c94be74cc24874c6feb5bb3c2152c398a18e/src/pip/_internal/utils/virtualenv.py#L14.
+            include: if self.prefix == self.base_prefix {
```

Comment:
Shouldn't this be `!=`, if the next branch is for venvs?

Reply:
No, the first branch is for venvs, right? See the linked pip source.

Reply:
Oh I got the condition wrong! The condition is inverted. I see.